aggregator/src/test/resources/application.properties

```diff
@@ -10,8 +10,22 @@ quarkus.datasource.username=postgres
 quarkus.datasource.password=postgres
 quarkus.datasource.jdbc.url=jdbc:postgresql://127.0.0.1:5432/notifications
 
+# configure your datasource
```

Comment:
This entire file should be removed.

You can add test configuration values to the `aggregator/src/main/resources/application.properties` file by using the `%test.` prefix.

Reply:
Okay, then this is different to Spring Boot. Didn't know that.